tests/snapshots/tests.contrib.pyramid.test_pyramid.test_simple_pyramid_app_endpoint.json

```diff
@@ -17,6 +17,7 @@
     "metrics": {
       "_dd.agent_psr": 1.0,
       "_dd.measured": 1,
+      "_dd.top_level": 1,
```

Comment:
is this a new snapshot file from upstream?

Reply:
Yes, I believe Zach generated this file while working on the pyramid escalation.